docs/editor/workspaces.md

```diff
@@ -18,25 +18,25 @@ The concept of a workspace enables VS Code to:
 * Store and restore UI state associated with that workspace (for example, the files that are opened).
 * Selectively enable or disable extensions only for that workspace.
 
-You may see the terms "folder" and "workspace" used interchangeably in VS Code documentation, issues, and community discussions. Think of a workspace as the root of a project that has extra VS Code knowledge and capabilities.
+You may see the terms "folder" and "workspace" used interchangeably in VS Code documentation, issues, and community discussions. You may also see the term "project" in VS Code documentation and in the documentation of other tools, such as IDEs, that VS Code users may also use.  VS Code does not have a definite concept of "project". However for the loose definition of "project" as a "logical collection of remote and local filesystem items, configuration, and settings", you can think of a workspace as the root of a project that has extra VS Code knowledge and capabilities.
 
 > **Note:** It is also possible to open VS Code without a workspace. For example, when you open a new VS Code window by selecting a file from your platform's **File** menu, you will not be inside a workspace. In this mode, some of VS Code's capabilities are reduced but you can still open text files and edit them.
 
 ## How do I open a VS Code "workspace"?
 
-The easiest way to open a workspace is using the **File** menu and selecting one of the available entries for opening. Alternatively you might also operate VS Code from a terminal and are able to just pass the path to a workspace to the `code` command for opening.
+The easiest way to open a workspace is using the **File** menu and selecting one of the available **Workspace** related entries for opening, such as **Open Workspace...** or **Open Folder...**. Alternatively you might also operate VS Code from a terminal and are able to just pass the path to a workspace as the first argument to the `code` command for opening.
```

Comment:
I had originally thought about explicitly spelling out "Open Folder" and "Open Workspace" but on macOS you will not find "Open Folder", only "Open" because there we can offer a single dialog to open both files and folders. That does not exist on Windows dialogs.